tests/assertions/ocp4/ocp4-pci-dss-4-0-4.12.yml

```diff
@@ -0,0 +1,352 @@
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-ocp-insecure-registries:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-ocp-no-ldap-insecure:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-openshift-api-server-audit-log-path:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-rbac-cluster-roles-defined:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-rbac-debug-role-protects-pprof:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-rbac-least-privilege:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-rbac-limit-cluster-admin:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-rbac-limit-secrets-access:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-rbac-pod-creation-access:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-rbac-roles-defined:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-rbac-wildcard-use:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-routes-protected-by-tls:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-scansettingbinding-exists:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-scc-drop-container-capabilities:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-container-allowed-capabilities:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-scc-limit-ipc-namespace:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-net-raw-capability:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-network-namespace:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-privilege-escalation:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-privileged-containers:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-process-id-namespace:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scc-limit-root-containers:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-scheduler-profiling-protected-by-rbac:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-scheduler-service-protected-by-rbac:
+   default_result: PASS
+   result_after_remediation: PASS
+ e2e-pci-dss-4-0-secrets-consider-external-storage:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-secrets-no-environment-variables:
+   default_result: MANUAL
+   result_after_remediation: MANUAL
+ e2e-pci-dss-4-0-security-profiles-operator-exists:
+   default_result: FAIL
+   result_after_remediation: FAIL
```

Comment:
Working on fixing the remediation for this in https://github.com/ComplianceAsCode/content/pull/12315